docs/development/malware-checks.rst

```diff
@@ -0,0 +1,109 @@
+Malware Checks
```

Comment:
I'll hitch this on @ewdurbin, but it _might_ be nice to also have a short section on hacking on `MalwareCheckBase` -- e.g., modifying `prepare` to propagate more parameters to `scan`. Thoughts?

Reply:
Part of me thinks that this documentation is already too long, but I'm happy to add a section if it would be valuable. Also interested in @ewdurbin's thoughts.

Reply:
Something simple along the lines of "Access to additional information for hooked checks" might be a good idea. This could indicate that checks that hook a `File` will receive a `file_url` kwarg and indicate that it comes from the `prepare` method of the MalwareCheckBase.

This helps document that `file_url` kwarg and also indicates how someone might add other dependencies.

Reply:
Sounds good!